src/proto/shared/event.proto

```diff
@@ -71,8 +70,7 @@ message ClusterEventReq {
 	RASEvent event = 2;	// RAS event.
 }
 
-// RASEventResp acknowledges receipt of an event notification and an error
-// status. Acknowledgment does not indicate that the event was handled.
+// RASEventResp acknowledges receipt of an event notification.
```

Comment:
Should be ClusterEventResp

Reply:
corrected in follow-up: https://jira.hpdd.intel.com/browse/DAOS-6436